vortex-array/src/array/varbinview/mod.rs

```diff
@@ -219,7 +220,25 @@ impl VarBinViewArray {
 
 impl ArrayFlatten for VarBinViewArray {
     fn flatten(self) -> VortexResult<Flattened> {
-        Ok(Flattened::VarBinView(self))
+        // TODO(aduffy): this flatten implementation is relatively expensive, allocating a new
+        //  VarBinArray, and decoding elements from this Array one-at-a-time for insertion.
+        //  In the future, this will be removed because we will provide VarBinViewArray as a
+        //  first-class Flattened variant with zero-copy to Arrow.
+        let mut builder = VarBinBuilder::<u32>::with_capacity(self.len());
```

Comment:
Is there a cast function in arrow-rs that will do this?

Reply:
i don't see any, may be looking in wrong place though https://github.com/apache/arrow-rs/blob/master/arrow-cast/src/cast/string.rs

Reply:
this https://github.com/apache/arrow-rs/blob/master/arrow-cast/src/cast/mod.rs#L1357?

Reply:
done